src/test/java/uk/gov/pay/products/client/publicapi/PublicApiRestClientTest.java

```diff
@@ -128,7 +131,7 @@ public void findPayment_shouldThrowAnException() {
                 .respondBadRequestWithBody(errorPayload);
 
         try {
-            publicApiRestClient.getPayment(paymentId);
+            publicApiRestClient.getPayment(apiToken, paymentId);
```

Comment:
Hoe about a test for getting 401 from PublicApi for an invalid api token?

Reply:
Yes, I have added the test. 👍